src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -239,6 +243,10 @@ export default {
 			type: Number,
 			default: 3
 		},
+		categoryPageName: {
```

Comment:
I think you'll need to add this prop to LoanCardController as well and then pass it on through to the `component` therein.

Reply:
This component is being used outside the controller, and the callouts aren't being added to the old cards, so I don't think the `LoanCardController` needs to be changed.